README.md

```diff
@@ -71,29 +71,52 @@ Helper functions will be written in due course to return all xpaths within a sch
 
 A given IATI codelist can be added to the schema. Example using the [Country](http://iatistandard.org/codelists/Country/) codelist.
 
-```
+```python
 import iati.core.default
 schema.codelists.add(iati.core.default.codelist('Country'))
 ```
 
 The default collection of IATI codelists can be added using:
 
-```
+```python
 import iati.core.default
-for _, codelist in iati.core.default.codelists().items():
+for codelist in iati.core.default.codelists().values():
     schema.codelists.add(codelist)
 ```
 
 ### Loading Rulesets
 
+The default IATI Ruleset can be loaded by using:
+
+```
+import iati.core.default
+
+iati.core.default.ruleset()
+```
+
+If you wish to load your own Ruleset you can do this using:
+
+```
+import iati.core.resources
+import iati.core.Rulesets
+
+ruleset_str = iati.core.resources.load_as_string(filepath)
```

Comment:
`iati.core.resources` is for this distributed as part of the library rather than user-defined data - this should use standard functionality under `os`.

Reply:
Fixed in 559eb1b38f93aebce0e4258e6942f3a1e2d15802